source/MaterialXView/Viewer.cpp

```diff
@@ -1025,6 +1031,30 @@ void Viewer::createAdvancedSettings(Widget* parent)
     });
     wedgeCountBox->set_value(8);
     wedgeCountBox->set_editable(true);
+
+    ng::Label* diagramLabel = new ng::Label(advancedPopup, "Diagram Output Options (D)");
```

Comment:
Hook up to MaterialXView. Note that the current dot output generally does nothing as it seems to only handle outputting functional graphs and the starting point is not that. I have not touched the previous API which is directly on the `Node` class , but it seems like it might be worthwhile to deprecate it.